docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer as composer
+WORKDIR /app
+COPY ./ /app
```

Comment:
What's actually needed to composer install? It may just be the `composer.{lock, json}` in which case restricting it will cause this layer to re-run less